examples/simple-non-wasm/src/main.rs

```diff
@@ -0,0 +1,217 @@
+extern crate alloc;
+use anyhow::Result;
+use fuel_executor::{
+    CustomHandler, CustomIndexExecutor, GraphQlApi, IndexerConfig, IndexerService, Manifest,
+    ReceiptEvent,
+};
+use fuel_indexer::EntityResult;
+use fuel_indexer_derive::graphql_schema;
+use fuel_indexer_schema::Address;
+use fuels::core::{abi_decoder::ABIDecoder, InvalidOutputType, ParamType, Token, Tokenizable};
+use serde::{Deserialize, Serialize};
+use std::path::PathBuf;
+use structopt::StructOpt;
+use tokio::join;
+use tracing::{error, info};
+use tracing_subscriber::filter::EnvFilter;
+
+#[derive(StructOpt)]
+#[structopt(
+    name = "Indexer Service",
+    about = "Standalone binary for the fuel indexer service"
+)]
+pub struct Args {
+    #[structopt(parse(from_os_str), help = "Indexer service config file")]
+    config: PathBuf,
+}
+
+// Load graphql schema
+graphql_schema!("simple_handler", "schema/counter.graphql");
+
+// Object being indexed
+#[derive(Deserialize, Serialize, Debug, Eq, PartialEq)]
+pub struct CountEvent {
+    count: u64,
+    timestamp: u64,
+}
+
+// Implement Tokenizable for object being indexed
+impl Tokenizable for CountEvent {
+    fn from_token(token: Token) -> Result<Self, InvalidOutputType> {
+        let arr: Vec<u8> = Vec::from_token(token)?;
+        let count = u64::from_be_bytes(arr[..8].try_into().unwrap());
+        let timestamp = u64::from_be_bytes(arr[8..].try_into().unwrap());
+
+        Ok(Self { count, timestamp })
+    }
+
+    fn into_token(self) -> Token {
+        Token::Array(vec![Token::U64(self.count), Token::U64(self.timestamp)])
+    }
+}
+
+// Implement Entity trait for object being indexed
+impl Entity for CountEvent {
```

Comment:
Ideally there would be macro to automagically derive `CountEvent` from GraphQL schema and make it implement `Entity` (not too versed in macros myself). 

Similar to what's in [derive/macro](https://github.com/FuelLabs/fuel-indexer/blob/master/derive/src/schema.rs)